quickstarts/microsoft.healthcareapis/workspaces/iotconnectors/azuredeploy.json

```diff
@@ -0,0 +1,209 @@
+            "dependsOn": [
+                "[variables('workspaceResourceId')]"
+            ],
+            "kind": "fhir-R4",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "authenticationConfiguration": {
+                    "authority": "[format('{0}{1}', environment().authentication.loginEndpoint, subscription().tenantId)]",
+                    "audience": "[format('https://{0}-fs-{1}.fhir.azurehealthcareapis.com', variables('workspaceName'), parameters('basename'))]",
+                    "smartProxyEnabled": false
+                }
+            }
+        },
+        {
+            "type": "Microsoft.HealthcareApis/workspaces/iotconnectors",
+            "apiVersion": "2022-05-15",
+            "name": "[variables('iotConnectorName')]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[variables('workspaceResourceId')]"
+            ],
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "ingestionEndpointConfiguration": {
+                    "eventHubName": "[variables('eventhubName')]",
+                    "consumerGroup": "$Default",
+                    "fullyQualifiedEventHubNamespace": "[format('en-{0}.servicebus.windows.net', parameters('basename'))]"
+                },
+                "deviceMapping": {
+                    "content": "[parameters('deviceMapping')]"
+                }
+            }
+        },
+        {
+            "type": "Microsoft.HealthcareApis/workspaces/iotconnectors/fhirdestinations",
+            "apiVersion": "2022-05-15",
+            "name": "[variables('fhirDestinationName')]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[variables('workspaceResourceId')]",
+                "[variables('iotConnectorResourceId')]"
+            ],
+            "properties": {
+                "resourceIdentityResolutionType": "Create",
+                "fhirServiceResourceId": "[variables('fhirServiceResourceId')]",
+                "fhirMapping": {
+                    "content": "[parameters('destinationMapping')]"
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Authorization/roleAssignments",
+            "apiVersion": "2020-08-01-preview",
+            "name": "[guid(format('{0}-FhirWriter', resourceGroup().id))]",
+            "dependsOn": [
+                "[variables('fhirServiceResourceId')]",
+                "[variables('iotConnectorResourceId')]"
+            ],
+            "properties": {
+                "roleDefinitionId": "[variables('fhirWriterRoleId')]",
+                "principalId": "[reference(variables('iotConnectorResourceId'), '2022-05-15', 'Full').identity.principalId]"
+            },
+            "scope": "[variables('fhirServiceResourceId')]"
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
Feedback addressed in new PR commit.